app/src/main/res/values/send_strings.xml

```diff
@@ -50,4 +50,14 @@
     <string name="send_switch_to_token">Tap to switch to %s</string>
     <string name="send_enter_amount">Enter Amount</string>
 
+    <string name="send_slider_text_format">Send %s %s</string>
+    <string name="send_max_warning_text_format">Max: %s %s</string>
+    <string name="send_min_warning_text_format">Min: %s %s</string>
+
+    <string name="send_fees">Fees:</string>
+    <string name="send_fees_format">Fees: %s %s</string>
```

Comment:
do we need to set $1%s or smth like this?

Reply:
no need here but we can do it if it is strictly 
for now when you have %s %s and $1%s $2%s getString(R.string. send_fees_format, "1", "2") will work same way